safe_core/src/client/mock/vault.rs

```diff
@@ -934,13 +916,13 @@ impl Vault {
                     .get_adata(append.address, requester_pk, request)
                     .and_then(move |data| match data {
                         AData::PubSeq(mut adata) => {
-                            unwrap!(adata.append(&append.values, index));
+                            unwrap!(adata.append(append.values, index));
```

Comment:
Can you make this change please?
```suggestion
                            adata.append(append.values, index)?;
```
`?` will return the error into the result which will be returned in the response.